duckduckgo/duckduckgo-privacy-extension
# Add child entity icons to the dashboard.

**Reviewer:**



## Description:
https://app.asana.com/0/0/1202836875374614/f

Adds icons for Google and Facebook child entities (added in the next blocklist update). The PR also fixes a bug with the display of the Facebook logo.

## Steps to test this PR:
1. Switch to the 'beta' blocklist (via devtools panel).
2. Visit test sites. Verify that for the new entities the correct icon is shown:
  - "Google Ads (Google)": youtube.com, android.com
  - "Google Analytics (Google)": blogger.com
  - "Youtube (Google)": https://privacy-test-pages.glitch.me/privacy-protections/youtube-click-to-load/ (in not blocked section).
  - "Instagram (Facebook)": https://www.cinemablend.com/movies/see-the-four-hobbits-from-lotr-reuniting-at-a-restaurant-and-totally-making-the-day-of-a-nearby-fan 

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
